### PR TITLE
Add a logo

### PR DESCRIPTION
**Pull request by jendiamond**
[feature/112400665](https://www.pivotaltracker.com/projects/1516637/stories/112400665)
Add a partial with a logo.

### CODE REVIEW

**Comment by jendiamond** [comment]
Not sure if you are testing views.

**Comment by thomasjung-sc** [comment]
Is it better to have this as a partial?

**Comment by thomas-jung** [comment]
Does this need to be a partial?

File: app/views/layouts/_logo.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<div id="logo">HIAKUS</div>
```
Comment (thomas-jung) [inline app/views/layouts/_logo.html.erb]: Need to fix typo and add a new line here

**Comment by thomas-jung** [comment]
:+1:

**Comment by thomas-jung** [comment]
Closing this one @jendiamond

**Pull request closed without merging**